sql/migrations/20230623143144_world.sql

```diff
@@ -0,0 +1,164 @@
+
+-- Shadowforge Cache
+UPDATE `gameobject` SET `spawntimesecsmin` = 0, `spawntimesecsmax` = 0 WHERE `guid` = 40694;
+
+-- Conspicuous Urn
+UPDATE `gameobject` SET `spawntimesecsmin` = 0, `spawntimesecsmax` = 0 WHERE `guid` = 40695;
+
+-- ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+-- Quests are Alliance / Horde Only
+UPDATE `quest_template` SET `RequiredRaces` = 77 WHERE `entry` IN (8951, 8952, 8953, 8954, 8955, 8956, 8958, 8959);
+UPDATE `quest_template` SET `RequiredRaces` = 178 WHERE `entry` IN (8957, 9016, 9017, 9018, 9019, 9020, 9021, 9022);
+
+-- ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+-- Correct Position of Altar of the Defiler
+UPDATE `gameobject` SET `position_x` = -11247.5, `position_y` = -2831.07, `position_z` = 159.971, `orientation` = 5.79449, `rotation2` = -0.241921, `rotation3` = 0.970296 WHERE  `guid`=16739;
+
+-- Add Missing Spawn Yell For Razelikh the Defiler
+INSERT INTO `creature_ai_events` (`id`, `creature_id`, `condition_id`, `event_type`, `event_inverse_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `action1_script`, `action2_script`, `action3_script`, `comment`) VALUES
+(766409, 7664, 0, 11, 0, 100, 0, 0, 0, 0, 0, 766409, 0, 0, 'Razelikh the Defiler - Yell on Spawn');
+INSERT INTO `creature_ai_scripts` (`id`, `delay`, `priority`, `command`, `datalong`, `datalong2`, `datalong3`, `datalong4`, `target_param1`, `target_param2`, `target_type`, `data_flags`, `dataint`, `dataint2`, `dataint3`, `dataint4`, `x`, `y`, `z`, `o`, `condition_id`, `comments`) VALUES
+(766409, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 4549, 0, 0, 0, 0, 0, 0, 0, 0, 'Razelikh the Defiler - Yell on Spawn');
+
+-- ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+-- Remove Incorrect Loot Template
+UPDATE `creature_template` SET `loot_id` = 0 WHERE `entry` = 235;
+DELETE FROM `creature_loot_template`  WHERE `entry` = 235;
+
+-- ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+-- Remove Incorrect Creature
+DELETE FROM `creature` WHERE `guid` = 21706;
+
+-- ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+-- Ran Bloodtooth's Skull Should Always Drop
+UPDATE `creature_loot_template` SET `ChanceOrQuestChance` = 100 WHERE `item` = 5388;
```

Comment:
Did you test this on classic to confirm it should be like that?

Reply:
See https://youtu.be/MMNpCZbWZ8c?t=547